packages/components/package.json

```diff
@@ -15,18 +15,26 @@
     "directory": "packages/components"
   },
   "type": "module",
-  "main": "dist/index.umd.js",
-  "module": "dist/index.es.js",
+  "main": "dist/index.umd.min.js",
+  "unpkg": "dist/index.umd.min.js",
+  "module": "dist/index.es.min.js",
+  "types": "es/components/index.d.ts",
   "exports": {
     ".": {
-      "import": "./dist/index.es.js",
-      "require": "./dist/index.umd.js",
-      "types": "./dist/components/index.d.ts"
-    }
+      "import": "./es/components/index.js",
+      "require": "./dist/index.umd.min.js",
+      "types": "./es/components/index.d.ts"
+    },
+    "./components/*": "./es/components/*.js",
+    "./components/*.js": "./es/components/*.js",
```

Comment:
I don't understand these lines. Why exporting `*` and `*.js` as `*.js`?

Reply:
fixed